app/main.py

```
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    for data in people:
+        person = Person(data["name"], data["age"])
+        persons.append(person)
+
+    for data in people:
+        person = Person.people[data["name"]]
+        if "wife" in data and data ["wife"] is not None:
+            person.wife = Person.people[data["wife"]]
+        if "husband" in data and data["husband"] is not None:
+            person.husband = Person.people[data["husband"]]
```

Related to the dict.get() and attribute-creation requirements in the description: ensure you only create `wife`/`husband` attributes when the corresponding dict value is truthy. Instead of `if "wife" in person_dict and person_dict['wife'] is not None: ...`, prefer `spouse_name = person_dict.get('wife')` and then `spouse_name and setattr(person, 'wife', Person.people[spouse_name])`. This keeps the attribute absent when the key is missing or the value is `None`, which is required by the description (accessing the attribute should raise AttributeError when missing) .